File: Cargo_Java_Backend/src/main/java/com/bdma/cargo/model/AddressModel.java
package com.bdma.cargo.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AddressModel {

    String id;
    String country;
    String city;
    String postalCode;
    String streetName;
    String streetNumber;
    String buildingNumber;
    String provence;

}

File: Flutter_App/lib/addressModel.dart

class AddressModel {
  late String id;
  String city;
  String country;
  String postalCode;
  String streetName;
  String streetNumber;
  String buildingNumber;
  String provence;

  AddressModel({
    required id,
    required this.city,
    required this.country,
    required this.postalCode,
    required this.streetName,
    required this.streetNumber,
    required this.buildingNumber,
    required this.provence,
  });

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'streetName': streetName,
      'streetNumber': streetNumber,
      'buildingNumber': buildingNumber,
      'provence': provence,
      'postalCode': postalCode,
      'city': city,
      'country': country,
    };
  }

  factory AddressModel.fromJson(Map<String, dynamic> json) {
    return AddressModel(
      id: json['id'],
      city: json['city'],
      country: json['country'],
      postalCode: json['postalCode'],
      streetName: json['streetName'],
      streetNumber: json['streetNumber'],
      buildingNumber: json['buildingNumber'],
      provence: json['provence'],
    );
  }
}